docs/data/data-grid/ai-assistant/AssistantWithDataSampling.js

```diff
@@ -0,0 +1,53 @@
+import * as React from 'react';
+import {
+  DataGridPremium,
+  unstable_gridDefaultPromptResolver as promptResolver,
+} from '@mui/x-data-grid-premium';
+import { useDemoData } from '@mui/x-data-grid-generator';
+
+const VISIBLE_FIELDS = [
+  'name',
+  'email',
+  'position',
+  'company',
+  'salary',
+  'phone',
+  'country',
+  'dateCreated',
+  'isAdmin',
+];
+
+function processPrompt(prompt, context, conversationId) {
+  return promptResolver(
+    'https://backend.mui.com/api/datagrid/prompt',
```

Comment:
I can hit it using Postman, should we disable cross-origin requests on it to avoid potential abuse?

<img width="400" alt="image" src="https://github.com/user-attachments/assets/ba48f4b2-6cae-4d38-86c9-a3d3f1943892" />